<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualizador DICOM</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(253, 247, 239);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1096</y>
        <width>755</width>
        <height>1772</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="main_vertical_layout">
       <item>
        <layout class="QHBoxLayout" name="top_buttons_layout">
         <item>
          <widget class="QPushButton" name="boton_cargar">
           <property name="font">
            <font>
             <family>Tw Cen MT Condensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(220, 187, 221);</string>
           </property>
           <property name="text">
            <string>Cargar DICOM</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="expandingSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="boton_volver">
           <property name="font">
            <font>
             <family>Tw Cen MT Condensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(220, 187, 221);</string>
           </property>
           <property name="text">
            <string>Volver</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layout_axial">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Tw Cen MT Condensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Corte Axial</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="campo_grafico_axial" native="true">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Tw Cen MT Condensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_axial">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layout_coronal">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Tw Cen MT Condensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Corte Coronal</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="campo_grafico_coronal" native="true">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Tw Cen MT Condensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_coronal">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layout_sagital">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Tw Cen MT Condensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Corte Sagital</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="campo_grafico_sagital" native="true">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>500</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Tw Cen MT Condensed</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_sagital">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
